skimage/metrics/simple_metrics.py

```diff
@@ -204,7 +204,7 @@ def normalized_mutual_information(image0, image1, *, bins=100):
 
     .. math::
 
-        Y(A, B) = \frac{H(A) + H(B)}{H(A, B)}
+       Y(A, B) = \frac{H(A) + H(B)}{H(A, B)}
```

Comment:
That doesn't fix it... I suspect an upstream issue. Does it render well for you locally?
The CI rendering is like mine locally, i.e.:

![still_broken](https://github.com/user-attachments/assets/15369da6-ba0d-4276-a8d2-35cc60663f8a)